tests/unit/HistogramTest.php

```diff
@@ -0,0 +1,49 @@
+<?php
+/**
+ * This file is part of graze/dog-statsd
+ *
+ * Copyright (c) 2016 Nature Delivered Ltd. <https://www.graze.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ *
+ * @license https://github.com/graze/dog-statsd/blob/master/LICENSE.md
+ * @link    https://github.com/graze/dog-statsd
+ */
+
+namespace Graze\DogStatsD\Test\Unit;
+
+use Graze\DogStatsD\Test\TestCase;
+
+class HistogramTest extends TestCase
+{
+    public function testHistogram()
+    {
+        $this->client->histogram('test_metric', 10);
+        $this->assertEquals('test_metric:10|h', $this->client->getLastMessage());
+
+        $this->client->histogram('test_metric', 1.2);
+        $this->assertEquals('test_metric:1.2|h', $this->client->getLastMessage());
+    }
+
+    public function testHistogramSample()
+    {
+        $this->client->histogram('test_metric', 5, 0.75);
```

Comment:
The sample rate here means it will only send a metric 75% of the time. This test does a single call which 25% of the time will fail (thus very flakey).

If you change the test to:

```php
        for ($i = 0; $i < 100; $i++) {
            $this->client->histogram('test_metric', $i, 0.75);
            $this->assertEquals("test_metric:{$i}|h|@0.75", $this->client->getLastMessage());
        }
```

It will fail.